docker-compose.yml

```
@@ -0,0 +1,15 @@
+services:
+  postgres:
+    image: postgres:15
```

```suggestion
    image: postgres:16
```

At least. 17 soon, https://www.postgresql.org/about/news/postgresql-17-rc1-released-2926/